opencti-platform/opencti-graphql/src/utils/filtering/filtering-utils.ts

```diff
@@ -222,16 +222,16 @@ const getConvertedRelationsNames = (relationNames: string[]) => {
  * - convert relation refs key if any
  * @param filterGroup
  */
-export const checkAndConvertFilters = (filterGroup?: FilterGroup | null) => {
+export const checkAndConvertFilters = (filterGroup?: FilterGroup | null, noFiltersChecking = false, authorizeNestedFiltersKeys = false) => {
```

Comment:
Please prefer using an `opts` parameter with different options inside, so we don't expand the arguments lists infinitely :)

Reply:
yes !!

Reply:
fixed